lib/intelipost/fluent_interface.rb

```diff
@@ -1,5 +1,22 @@
 module Intelipost
   module FluentInterface
+
+    def self.included(cls)
+      cls.extend(ClassMethods)
```

Comment:
@sgabler it's a hook method. [self.included](https://rubymonk.com/learning/books/4-ruby-primer-ascent/chapters/48-advanced-modules/lessons/117-included-and-extend)
I read a lot of Dave Thomas metaprogramming talks/books, that's why I used `cls` instead of `klass`. =P